src/assets/usage-examples-code/sprites/free_sprite/free_sprite-3-free-sprite.cs

```diff
@@ -0,0 +1,45 @@
+/*
+    Usage Example: freeing a sprite
+*/
+
+using System;
+using SplashKitSDK;
+
+public class Program
```

Comment:
All C# programs on splashkit must be done using top level statements only. Can you update this to not use public class or static void.

Reply:
I have made them top level